opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/aggregation/AggregationQueryBuilder.java

```diff
@@ -112,10 +112,24 @@ public Map<String, OpenSearchDataType> buildTypeMapping(
       List<NamedAggregator> namedAggregatorList,
       List<NamedExpression> groupByList) {
     ImmutableMap.Builder<String, OpenSearchDataType> builder = new ImmutableMap.Builder<>();
-    namedAggregatorList.forEach(agg -> builder.put(agg.getName(),
-        OpenSearchDataType.of(agg.type())));
-    groupByList.forEach(group -> builder.put(group.getNameOrAlias(),
-        OpenSearchDataType.of(group.type())));
+    namedAggregatorList.forEach(agg -> {
+      if (agg.type() == ExprCoreType.TIMESTAMP
+          || agg.type() == ExprCoreType.DATETIME) {
+        // TODO: check for DATE and TIME when it is supported
```

Comment:
remove TODO
We would never reach here with a DATE or TIME since these types would have a OpenSearchDateType defined.